src/renderers/webgl/WebGLCapabilities.js

```diff
@@ -0,0 +1,57 @@
+THREE.WebGLCapabilities = function( gl, extensions, parameters ){
+    
+    this.getMaxPrecision = function ( precision ) {
+    
+    	if ( precision === 'highp' ) {
+    
+    		if ( gl.getShaderPrecisionFormat( gl.VERTEX_SHADER, gl.HIGH_FLOAT ).precision > 0 &&
+    		     gl.getShaderPrecisionFormat( gl.FRAGMENT_SHADER, gl.HIGH_FLOAT ).precision > 0 ) {
+    
+    			return 'highp';
+    
+    		}
+    
+    		precision = 'mediump';
+    
+    	}
+    
+    	if ( precision === 'mediump' ) {
+    
+    		if ( gl.getShaderPrecisionFormat( gl.VERTEX_SHADER, gl.MEDIUM_FLOAT ).precision > 0 &&
+    		     gl.getShaderPrecisionFormat( gl.FRAGMENT_SHADER, gl.MEDIUM_FLOAT ).precision > 0 ) {
+    
+    			return 'mediump';
+    
+    		}
+    
+    	}
+    
+    	return 'lowp';
+    
+    };
+
+	// GPU capabilities
+	this.precision = parameters.precision !== undefined ? parameters.precision : 'highp',
+	this.logarithmicDepthBuffer = parameters.logarithmicDepthBuffer !== undefined ? parameters.logarithmicDepthBuffer : false;
```

Comment:
This seems to be weird in that it isn't a queried WebGLCapability like everything else.  Why is every other variable queried and this one isn't.